jest-lint.config.js

```diff
@@ -0,0 +1,7 @@
+const baseConfig = require('./jest-base.config.js');
+
+module.exports = Object.assign({}, baseConfig, {
+    runner: 'jest-runner-eslint',
+    displayName: 'lint',
+    testMatch: ['<rootDir>/src/**/*.(ts|js)'],
```

Comment:
Can this move into `jest-base.config.js` as well?

Reply:
Well for test we want it to have specific, but lint we want all files. We can move it, but I find this more explicit.

Reply:
Ooooops! Totally missed the `.spec` part. My bad. 🤦🏾‍♂️